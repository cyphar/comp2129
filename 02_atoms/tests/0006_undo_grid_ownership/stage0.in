START 2 6 6
PLACE 0 0
PLACE 0 1
PLACE 1 1
PLACE 0 1
PLACE 1 1
PLACE 5 5
DISPLAY
PLACE 0 0
PLACE 1 0
DISPLAY
UNDO
PLACE 5 0
PLACE 1 0
DISPLAY
