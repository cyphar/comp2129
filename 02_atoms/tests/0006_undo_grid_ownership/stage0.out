Game Ready
Red's Turn

Green's Turn

Red's Turn

Green's Turn

Red's Turn

Green's Turn

Red's Turn


+-----------------+
|R1|  |  |  |  |  |
|G2|R2|  |  |  |  |
|  |  |  |  |  |  |
|  |  |  |  |  |  |
|  |  |  |  |  |  |
|  |  |  |  |  |G1|
+-----------------+

Green's Turn

Cannot Place Atom Here


+-----------------+
|R1|R1|  |  |  |  |
|  |R3|  |  |  |  |
|R1|  |  |  |  |  |
|  |  |  |  |  |  |
|  |  |  |  |  |  |
|  |  |  |  |  |G1|
+-----------------+

Red's Turn

Green's Turn

Red's Turn


+-----------------+
|R1|G1|  |  |  |R1|
|G2|R2|  |  |  |  |
|  |  |  |  |  |  |
|  |  |  |  |  |  |
|  |  |  |  |  |  |
|  |  |  |  |  |G1|
+-----------------+

